app/javascript/components/applications/application_form/AddressForm.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react'
-import { Text } from 'react-form'
+import { FieldWrapper } from '~/utils/form/final_form/Field'
 import { isEmpty } from 'lodash'
 import { maxLengthMap } from '~/utils/formUtils'
 
-const buildField = (memberType, nestedField, fieldMap, fieldKey) => {
+const buildField = (memberType, name, fieldMap, fieldKey) => {
```

Comment:
@akegan could we update the variable `name` to have a slightly more descriptive name? i'm not too familiar with the paper app form so i'm not sure what `name` is here. same for use of `name` in `AddressForm` component below.

Reply:
looks like name is used in the case of FieldArrays to pass down what the name is for form item in the value in the array. 

For example. for a household member, we'd expect it to be e.g. "household_members[0]", then the e.g. first name field would have the value "household_members[0].first_name" as it's "name". Maybe fieldName would make more sense?